examples/s3d/test_s3d.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Let's get some feedback about how we should structure test folder for examples. Tentatively, this could be:
```
examples/my_project/
    tests/
        test_module_1.py
        test_module_2.py
    module_1.py
    module_2.py
```